packages/block-library/src/navigation-menu/edit.js

```diff
@@ -108,14 +113,70 @@ function NavigationMenu( {
 		}
 	};
 
-	useEffect( () => {
-		// Set/Unset colors CSS classes.
-		setAttributes( {
-			backgroundColorCSSClass: backgroundColor.class ? backgroundColor.class : null,
-			textColorCSSClass: textColor.class ? textColor.class : null,
-		} );
-	}, [ backgroundColor.class, textColor.class ] );
+	const handleCreateEmpty = () => {
+		setBlocksTemplate( null );
+		setInitialPlaceholder( false );
+	};
+
+	const handleCreateFromExisting = () => {
+		setBlocksTemplate( defaultPagesMenuItems );
+		setInitialPlaceholder( false );
+	};
+
+	//
+	// MARKUP
+	//
+
+	// Build Inline Styles
+	const navigationMenuInlineStyles = {
+		...( textColor && { color: textColor.color } ),
+		...( backgroundColor && { backgroundColor: backgroundColor.color } ),
+	};
+
+	// Build ClassNames
+	const navigationMenuClasses = classnames(
+		'wp-block-navigation-menu', {
+			'has-text-color': textColor.color,
+			'has-background-color': backgroundColor.color,
+			[ attributes.backgroundColorCSSClass ]: attributes && attributes.backgroundColorCSSClass,
+			[ attributes.textColorCSSClass ]: attributes && attributes.textColorCSSClass,
+		}
+	);
 
+	// UI State: initial placeholder
+	if ( ! hasExistingNavItems && initialPlaceholder ) {
+		return (
+			<Placeholder
+				className="wp-block-navigation-menu-placeholder"
+				icon="menu"
+				label={ __( 'Navigation Menu' ) }
+			>
+				<p className="wp-block-navigation-menu-placeholder__instructions">
```

Comment:
Placeholder has an `instructions` prop that'll handle rendering this.

Reply:
Silly of me to have missed this as I was taking note of the props on that component. Ironic that my classname of `___instructions` matches up the nomenclature the component author used for the prop!

Anyway, I've resolved this. Thanks!